utils/db/repos.ts

```diff
@@ -170,3 +170,50 @@ export const getRepoConfigByUserAndRepo = async (provider: string, repoName: str
 	}
 	return userRows[0].config;
 }
+
+export const insertRepoConfig = async (userId: string, repoIds: number[]) => {
+	const query = `
+		INSERT INTO repo_config (repo_id, user_id, auto_assign, comment_setting)
+		SELECT id, $1, true, true
+		FROM unnest($2::INT[]) AS t(id)
```

Comment:
What does `t(id)` do?

Reply:
it's a derived table t from the items in array and we can use t(id) to specify column name for the derived table. but here it does nothing.